azure-auth-helper/src/test/java/com/microsoft/azure/auth/AzureAuthHelperTest.java

```diff
@@ -61,12 +61,8 @@ public void testRefreshTokenInvalidParameter() throws Exception {
 
     @Test
     public void tetGetAzureSecretFile() throws Exception {
-        File azureSecretFile = AzureAuthHelper.getAzureSecretFile();
-        assertEquals(Paths.get(System.getProperty(Constants.USER_HOME_KEY), ".azure", "azure-secret.json").toString(),
-                azureSecretFile.getAbsolutePath());
-        System.setProperty(Constants.AZURE_HOME_KEY, "test_dir");
-        azureSecretFile = AzureAuthHelper.getAzureSecretFile();
-        assertEquals(Paths.get("test_dir", "azure-secret.json").toFile().getAbsolutePath(),
+        final File azureSecretFile = AzureAuthHelper.getAzureSecretFile();
```

Comment:
do we need to ensure `System.getenv(Constants.AZURE_CONFIG_DIR)` is empty or add correspond test case?

Reply:
java cannot set env itself, there is a workaround: https://stackoverflow.com/questions/318239/how-do-i-set-environment-variables-from-java

Reply:
test case updated.